nixos/modules/services/networking/tailscale.nix

```diff
@@ -9,6 +9,13 @@ in {
   options.services.tailscale = {
     enable = mkEnableOption "Tailscale client daemon";
 
+    openFirewall = mkOption {
+      type = types.bool;
+      default = true;
```

Comment:
the default for these options should be `false` as outlined in #149889 :confused:

Reply:
```suggestion
      default = false;
```